### ISSUE TITLE
Error after updating from 1.9.10 to 1.9.17

### ISSUE THREAD

**Issue by jokandre** [taye/interact.js#824]
I decided to update my app that was already using InteractJS without issues, now after build it fills my console with these errors. I only copied/pasted 3 as an example. My guess is that it is something to do with typescript. 

Any help would be appreciated.

```
ERROR in /.../node_modules/@interactjs/types/index.d.ts(719,138):                             nuxt:typescript 17:34:11
719:138 Namespace 'unknown' has no exported member 'Element'.
    717 |          * @return {function | object} The checker function or this Interactable
    718 |          */
  > 719 |         rectChecker(checker: (element: import("@interactjs/types/index").Element) => any): this | ((element: import("@interactjs/types").Element) => {
        |                                                                                                                                          ^
    720 |             left: number;
    721 |             right: number;
    722 |             top: number;


 ERROR  ERROR in /.../node_modules/@interactjs/types/index.d.ts(754,48):                              nuxt:typescript 17:34:11
754:48 Namespace 'unknown' has no exported member 'Context'.
    752 |          * @return {Node} The context Node of this Interactable
    753 |          */
  > 754 |         context(): import("@interactjs/types").Context;
        |                                                ^
    755 |         inContext(element: Document | Node): boolean;
    756 |         testIgnoreAllow(this: Interactable, options: {
    757 |             ignoreFrom?: IgnoreValue;


 ....
 ERROR  ERROR in /.../node_modules/@interactjs/types/index.d.ts(3461,65):                             nuxt:typescript 17:34:15
3461:65 Namespace 'unknown' has no exported member 'Point'.
    3459 |                     closest?: any;
    3460 |                     targetFields?: string[][];
  > 3461 |                 }, any>>) => void | import("@interactjs/types").Point;
         |                                                                 ^
    3462 |                 stop: (arg: import("@interactjs/modifiers/base").ModifierArg<import("@interactjs/modifiers/base").ModifierState<import("@interactjs/modifiers/snap/pointer").SnapOptions, {
    3463 |                     offsets?: import("@interactjs/modifiers/snap/pointer").Offset[];
    3464 |                     closest?: any;


 ERROR  ERROR in /.../node_modules/@interactjs/types/typings.d.ts(1,27):                              nuxt:typescript 17:34:15
1:27 File '/home/machine/Documents/Projects/manga_learner/manga_admin/node_modules/@interactjs/types/index.d.ts' is not a module.
  > 1 | import * as Interact from './index'
      |                           ^
    2 | 
    3 | export as namespace Interact
    4 | export = Interact
```

### System configuration
**interact.js version**: 1.9.17 
** typescript version**: 3.8
**Operating System**: Linux

**Comment by jokandre** [comment]
I imported interactJS the following way:
```
import interact from "interactjs";
```
and did NOT install @interactjs/types. Are they a requirement now?

**Comment by taye** [comment]
I've released `v1.9.18` which hopefully resolves this issue.

**Comment by jokandre** [comment]
@taye Thanks for the quick feedback. It did solve the issue.
